astropy/cosmology/io/tests/test_row.py

```diff
@@ -113,7 +113,7 @@ def test_is_equivalent_to_row(self, cosmo, to_format, format):
         assert not isinstance(obj, Cosmology)
 
         is_equiv = cosmo.is_equivalent(obj, format=format)
-        assert is_equiv is (True if format is not False else False)
+        assert is_equiv is (bool(format is not False))
```

Comment:
same here: no need for `bool` conversion

Reply:
✅ Done.